website/docs/faqs/Accounts/change-users-license.md

```diff
@@ -0,0 +1,19 @@
+---
+title: How do I change a user license type to read-only in dbt Cloud?
+description: "Changing a user license type to read-only in dbt Cloud"
+sidebar_label: 'How to change a user license type to read-only'
+id: change-user-license
+
+---
+
+This article outlines the steps for changing the license type for a user from `developer` to `read-only` in dbt Cloud. This is frequently done to free up a billable seat but retain the user’s access to view the information in the dbt Cloud account. You must be an account owner/admin to perform these actions.
```

Comment:
```suggestion
You might want to change the user license type from `developer` to `read-only` in dbt Cloud. It's commonly done to free up a billable seat but retain the user’s access to view the information in the dbt Cloud account. You must be an account owner or have admin privileges to perform these actions.
```

Reply:
sugg: for a more conversational tone

Reply:
@nghi-ly This paragraph was created based on feedback from @runleonarun that the article needed an introductory statement, so I'd like to get her feedback on the changes.  It starts a little abruptly -how about "The following instructions detail the steps for changing a user license from `developer` to `read-only`. You be the account owner or have admin privileges to perform these actions"

If the statement about the permissions is last, it's more likely to stick in the user's mind and reduce the chance they skip over it. We can trim the fat by removing the "billable seat" bit.

Reply:
@matthewshaver / @runleonarun :  i'm super pro intro statements! they can help with SEO (depending what our infra is)!

for a "How do I ... ? " page, i don't think there's a lot of value with: "The following instructions detail the steps for changing a user license from developer to read-only." or "This article outlines the steps for changing the license type for a user from `developer` to `read-only` in dbt Cloud."

it's almost like repeating the title but turning the question into a sentence, which isn't adding anything new or relevant.

Reply:
I think we should go with @nghi-ly 's edits, while making the last bit active: "It's commonly done to" to "You might make this change to free up..." Like this:
```suggestion
To change the license type for a user from `developer` to `read-only` in dbt Cloud, you must be an account owner or have admin privileges. You might make this change to free up a billable seat but retain the user’s access to view the information in the dbt Cloud account.
```